src/main/java/com/ibm/watson/developer_cloud/text_to_speech/v1/Customizations.java

```diff
@@ -0,0 +1,226 @@
+   * 
+   * @param model
+   *          the model to be saved
+   * @return a reference to the given CustomVoiceModel. If a new
+   *         CustomVoiceModel is created, the id field will be populated.
+   */
+  public ServiceCall<CustomVoiceModel> saveVoiceModel(final CustomVoiceModel model) {
+    final boolean isNew = model.getId() == null;
+    final String path = isNew ? PATH_CUSTOMIZATIONS : String.format(PATH_CUSTOMIZATION, model.getId());
+
+    final RequestBody body = RequestBody.create(HttpMediaType.JSON, model.toString());
+    final Request request = RequestBuilder.post(path).body(body).build();
+
+    return service.createNewServiceCall(request, new ResponseConverter<CustomVoiceModel>() {
+      @Override
+      public CustomVoiceModel convert(Response response) {
+        CustomVoiceModel newModel = ResponseUtils.getObject(response, CustomVoiceModel.class);
+
+        if (newModel != null && newModel.getId() != null) {
+          model.setId(newModel.getId());
+        }
+
+        return model;
+      }
+    });
+  }
+
+  /**
+   * Deletes the given CustomVoiceModel (requires a valid id to be set).
+   */
+  public ServiceCall<Void> deleteVoiceModel(CustomVoiceModel model) {
+    Validator.notEmpty(model.getId(), "model id must not be empty");
+
+    final Request request = RequestBuilder.delete(String.format(PATH_CUSTOMIZATION, model.getId())).build();
+    return service.createNewServiceCall(request, ResponseConverterUtils.getVoid());
+  }
+
+  /**
+   * Gets all customized word translation of the given CustomVoiceModel.
+   * 
+   * @param model
+   *          the VoiceModel
+   * @return a list of all CustomTranslations
+   */
+  public ServiceCall<List<CustomTranslation>> getWords(CustomVoiceModel model) {
+    Validator.notNull(model.getId(), "model id can not be null");
+
+    final Request request = RequestBuilder.get(String.format(PATH_WORDS, model.getId())).build();
+    ResponseConverter<List<CustomTranslation>> converter = ResponseConverterUtils
+        .getGenericObject(TYPE_CUSTOM_TRANSLATIONS, WORDS);
+    return service.createNewServiceCall(request, converter);
+  }
+
+  /**
+   * Saves or updates a custom word translation. If no translation with the
+   * given word is existing, a new translation is created. Elsewise, the
+   * existing translation is updated.
+   * 
+   * @param model
+   *          the VoiceModel
+   * @param translation
+   *          the translation to be saved or updated
+   * @return
+   */
+  public ServiceCall<Void> saveWord(CustomVoiceModel model, CustomTranslation translation) {
+    return saveWords(model, Arrays.asList(translation));
+  }
+
+  /**
+   * Saves or updates multiple custom word translations.
+   * 
+   * @param model
+   *          the CustomVoiceModel
+   * @param translations
+   *          the translations to be saved or updated
+   * @return
+   */
+  public ServiceCall<Void> saveWords(CustomVoiceModel model, List<CustomTranslation> translations) {
```

Comment:
You can combine this method with `saveWord(CustomVoiceModel model, CustomTranslation translation)` using 3 dots in the method signature.

`saveWord(CustomVoiceModel model, CustomTranslation ...translation)`

users will be able to call it with one or more `CustomTranslation`s

Internally you handle it as an array